install_files/ansible-base/roles/restrict-direct-access/tasks/fetch_tor_config.yml

```diff
@@ -5,6 +5,7 @@
     path: "{{ tor_hidden_services_parent_dir }}/{{ item.service }}/hostname"
     delay: 5
   with_items: "{{ tor_instances }}"
+  when: "v2_onion_services|default(True)"
```

Comment:
Rather than repeating `|default(True)`, set `v2_onion_services: True` in `install_files/ansible-base/group_vars/all/securedrop`.

Reply:
I will do this.